easybuild/easyconfigs/j/Java/Java-11.0.6-ppc64le.eb

```diff
@@ -0,0 +1,15 @@
+name = 'Java'
+version = '11.0.6'
```

Comment:
According to https://wiki.openjdk.java.net/display/JDKUpdates/JDK11u there is no 11.0.6 yet?

Reply:
I've misread the SAP releases page and picked up a pre-release by mistake. That will be easy enough for me to fix.

Reply:
Changed to released version